src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -555,33 +555,38 @@ private void AdjustPathEndpoints()
     }
 
     /// <summary>
-    ///   Get intersection count and priority for sorting.
+    ///   Calculate priority of a path for sorting.
     /// </summary>
     /// <remarks>
     ///   <para>
-    ///     Note that priority should be within (-10, 10), and the return value is relative.
+    ///     Priority should be within (-10, 10)
     ///   </para>
     /// </remarks>
-    private int IntersectionCountWithPriority((Vector2[] Path, int Priority) pathPriorityTuple)
+    /// <returns>
+    ///   Calculated priority: 8bit region intersection; 8bit path intersection;
```

Comment:
```suggestion
    ///   Calculated priority: 8-bit region intersection; 8-bit path intersection;
```
either that way or "8 bit" is the way bitness should be written.